### DISCUSSION TITLE
Compass and map orientation

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#691] [Q&A] [answered]
The introduction of the compass-based location direction indicator was a big help for orientation while hiking. You always see at a glance where your smartphone is pointing towards. Thanks again!
For me it is a little bit confusing that the compass at the top left is moving as well and is showing the direction of the phone. I would expect to indicate the map orientation, and that is (normaly) north or you are rotating the map manually. The orientation below the compass should show the smartphone's direction (N, S, NW,...) but the compass should indicate the map orientation.
What is your opinion?

**Comment by kds112** [comment]
The effect only occurs in normal map view with a northern directed map. If the map is rotated, the compass shows the map direction correctly.

**Reply by WolfgangZ42** [reply]
> with a northern directed map

you don't need a compass needle for the map orientation. Right?

**Reply by kds112** [reply]
The "problem" is that you don't see whether the map is rotated and the compass shows the map orientation OR the map is north-orientated and the compass shows the phone orientation. There is no indication, you have to know it...

**Comment by devemux86** [comment, marked as answer]
Compass displays:

- Map north when map rotation is enabled in navigation and follow modes
(that's how it works all these years)

- True north when map rotation is disabled

- Compass label displays the direction relative to compass

Map rotation is only useful in navigation and follow modes.
Not in normal map view, if it's confusing we could disable it.

**Reply by kds112** [reply]
No, please just leave it as it is. Thanks 👍

**Reply by MotoUKRider** [reply]
If I were walking, I may find a use for being able to rotate a map in the direction that I was heading, manipulate the view and work out *where the heck I was going to/from* :mag_right:

**Reply by devemux86** [reply]
Do you find the above [description](https://github.com/devemux86/cruiser/discussions/691#discussioncomment-11613973) of how the compass works is good enough?
Is there anything else that needs to be clarified?

**Reply by kds112** [reply]
It is clear. Thanks👍
